src/dispatchers/keyEventDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export class KeyEvent {
```

Comment:
But there are multiple types of Key events (`keydown`, `keyup`, etc), and I'm rolling up the different types of mouse events. Also serves to prevent confusion between "key" as in keyboard and a dictionary "key".